README.md

```diff
@@ -1,7 +1,128 @@
 # KerasCV
-KerasCV is a repository of modular building blocks (layers, metrics, losses, data-augmentation) that applied computer vision engineers can leverage to quickly assemble production-grade, state-of-the-art training and inference pipelines for common use cases such as image classification, object detection, image segmentation, image data augmentation, etc. 
+KerasCV is a repository of modular building blocks (layers, metrics, losses, data-augmentation) that
+applied computer vision engineers can leverage to quickly assemble production-grade, state-of-the-art
+training and inference pipelines for common use cases such as image classification, object detection, 
+image segmentation, image data augmentation, etc. 
 
-KerasCV can be understood as a horizontal extension of the Keras API: they're new first-party Keras objects (layers, metrics, etc) that are too specialized to be added to core Keras, but that receive the same level of polish and backwards compatibility guarantees as the rest of the Keras API and that are maintained by the Keras team itself (unlike TFAddons).
+KerasCV can be understood as a horizontal extension of the Keras API: they're new first-party Keras 
+objects (layers, metrics, etc) that are too specialized to be added to core Keras, but that receive 
+the same level of polish and backwards compatibility guarantees as the rest of the Keras API and that 
+are maintained by the Keras team itself (unlike TFAddons).
 
-## Under Construction
-KerasCV is a relatively new project.  Expect regular updates and changes in the near future.
+# No Goals
+- **KerasCV is not a repository of blackbox end-to-end solutions (like TF Hub or ModelGarden).**
+
+    For the time being, it is focused on modular and reusable building blocks. We expect models in 
+    ModelGarden or TF Hub to eventually be built on top of the building blocks provided by KerasCV.
+
+    In the process of developing these building blocks, we will by necessity implement end-to-end 
+    workflows, but they're intended purely for demonstration and grounding purposes, they're not 
+    our main deliverable.
+
+
+- **KerasCV is not a repository of low-level image-processing ops, like tf.vision.** 
+
+    KerasCV is fundamentally an extension of the Keras API: it hosts Keras objects, like layers, 
+    metrics, or callbacks. Low-level C++ ops should go elsewhere.
+
+
+- **KerasCV is not a research library.**
```

Comment:
I think partially we are trying to defining what is this more concretely at:

https://github.com/keras-team/keras-cv/discussions/52#discussioncomment-2058663

In a field like AI/Deep learning it is a little bit hard to talk about  the boundaries between `industry-strength versions` and research in many cases.

Where to put this boundaries is partially opinionated of course but I still think that we could find a baseline to share with the community to not arbitrarily discriminate too much when we need to reject a contribution proposal.
Using only the classic feedback "we have limited" resources it is generally a little bit too weak especially if plan to scale over the community co-ownership.

Reply:
Yeah, this is a pretty nuanced point.

In particular, I believe we mean the focus is on practical components for applied ML.  This doesn't mean researchers won't use our preprocessing implementations or modelling implementations, they are just not the people we specifically consider when we are authoring the library.

Reply:
Yes probably this is a little bit better. I think that we need to explain this in term o balance other then bold claims as I suppose we still want that researchers could use this also in the worst case scenario when it will be only used in paper only for comparing over strong and well known reproducible baselines.

Reply:
Yeah, this is probably too strongly worded.  Perhaps we can get the point across by moving this to goals and instead stating that our target is applied ML.